src/verify.c

```diff
@@ -263,7 +263,9 @@ static int get_required_files(struct manifest *official_manifest, struct list *s
 
 	progress_set_next_step("download_fullfiles");
 	print("\n");
+	progress_set_spinner(true);
```

Comment:
why are you setting the spinner here? we do have a progress report for downloading fullfiles

Reply:
I have slightly changed the implementation. Not setting the spinner anymore because it wont use curl sync download feature. so the spinner wont work. Instead I am thinking of implementing this in the form of reporting an indeterminate initially.